docs/internals/network/basic_networking/basic_networking.md

```diff
@@ -0,0 +1,250 @@
+# Basic Networking
+
+In this part we will go through some of basic Networking Concepts that is very related to how networking in zos working
+
+## Components
+
+- Namespaces: As you may know linux have the concept of Namespaces which includes (users, net, ....) namespaces. which is basically considered as a full copy of the network stack that is fully isolated from the network stack of the machine
+- Bridge: A network bridge is a virtual network device that forwards packets between two or more network segments. A bridge behaves like a virtual network switch and works transparently.
+- veth: Think of it as a piece of wire (cable) that connects two ends
+
+## What we are going to achieve in this doc
+
+- we will create 3 namespaces we are going to make connectivity between them using bridges and some routing rules
+- bridges works on layer2 only so it can not do routing so it will just broadcast the packets to the connected nodes
```

Comment:
We aim to create three network namespaces and establish connectivity between them using bridges and routing rules. Bridges operate at Layer 2, so they only broadcast packets to connected nodes without routing.